app/model/ModelV2.scala

```diff
@@ -20,37 +19,38 @@ object JourneyDataV2 {
   implicit val format: Format[JourneyDataV2] = Json.format[JourneyDataV2]
 }
 
-case class JourneyConfigV2(
-                            version: Int,
-                            options: JourneyOptions,
-                            labels: Option[JourneyLabels] = None
+case class JourneyConfigV2(version: Int,
+                           options: JourneyOptions,
+                           labels: Option[JourneyLabels] = None
                           )
 
 case class ResolvedJourneyConfigV2(journeyConfig: JourneyConfigV2) {
   val version: Int = journeyConfig.version
   val options: ResolvedJourneyOptions = ResolvedJourneyOptions(journeyConfig.options)
-  val labels: ResolvedJourneyLabels = ResolvedJourneyLabels(journeyConfig.labels.getOrElse(JourneyLabels()), options.phaseFeedbackLink)
+  val isWelsh: Option[Boolean] = journeyConfig.labels map {
```

Comment:
this could be a flatMap and contains instead to return a boolean which wouldnt need the get or else below